simapp/params/amino.go

```diff
@@ -0,0 +1,23 @@
+package params
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/codec/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+)
+
+// MakeEncodingConfig creates an EncodingConfig for an amino based test configuration.
+//
+// TODO: this file should add a "+build test_amino" flag for #6190 and a proto.go file with a protobuf configuration
+func MakeEncodingConfig() EncodingConfig {
```

Comment:
I'm not sure that simapp is the right place to place this function what about moving it to `codec/` or `codec/types`?

Reply:
Well the function does need to occur in simapp because it is specific to the SDK's test setup. But `EncodingConfig` itself can live outside of simapp maybe, but it really is test specific. It's not a core type.